src/Toggler.js

```diff
@@ -31,10 +31,13 @@ class Toggler extends State {
 
 	/**
 	* Manually collapse the content's visibility.
-	* @param {!Element} header
+	* @param {string|!Element} header
 	*/
 	collapse(header = this.header) {
-		var content = this.getContentElement_(header);
+		let content = this.getContentElement_(header);
+		if (!core.isElement(header)) {
+			header = this.getHeaderElements_(header);
+		}
```

Comment:
I see this lines are repeated here and inside `expand()`. Maybe we could abstract that into `getHeaderElements_` ?